Apps/W1/HybridGP/app/src/pages/HybridCloudWizardExtension.PageExt.al

```diff
@@ -73,9 +73,27 @@ pageextension 4014 "Hybrid Cloud Wizard Extension" extends "Hybrid Cloud Setup W
         {
             trigger OnAfterAction()
             begin
-                if Rec."Product ID" = 'DynamicsGP' then
+                if Rec."Product ID" = 'DynamicsGP' then begin
+                    CleanupSettingsForCompaniesPreviouslyMigrated();
                     Page.Run(Page::"GP Migration Configuration");
+                end;
             end;
         }
     }
+
+    local procedure CleanupSettingsForCompaniesPreviouslyMigrated()
```

Comment:
This is for partners who are migrating multiple companies, one at a time. When they complete a migration for company 'One', and then they go to configure the migration for company 'Two', the previously migrated company 'One' is still in the list and is confusing since it is already migrated.